### PR TITLE
feat(backgroundTranslation): serviceMessage

### LINKED ISSUE #909: Background string aren't translated
**Describe the bug**
Some strings aren't translated.

**Extension**
[3.71.0](https://github.com/dis-moi/extension/commit/6ca536f7cac1f9679dfb9afdb9641d184cd043c9)

**Additional context**
See for example:
- https://github.com/dis-moi/extension/blob/master/src/app/background/sagas/serviceMessage.saga.ts#L34-L38
- "Sources d'informations" in `background/index.ts`

### PR DESCRIPTION
**Pull request by gregoirelacoste**
closes #909

### CODE REVIEW

**Comment by gregoirelacoste** [comment]
@MaartenLMEM are you ok with this translation ? https://github.com/dis-moi/extension/pull/945/files#diff-7fb75433852be48976b2d6861e57933351a3f1f0344f291de0307c2f9d1c2479R137

**Comment by MaartenLMEM** [comment]
will be reviewed during our next meeting with @prk-dismoi

**Review by lutangar — APPROVED** [review]
This is mostly wording and looks ok as far as I'm concerned, @MaartenLMEM I'll let you give the final word.

File: src/i18n/resources/en/extension.json [hunk 1]
```diff
@@ -133,5 +133,14 @@
     "tos": "/cgu",
     "about": "/a-propos",
     "uninstall": "/desinstaller"
+  },
+  "services": {
+    "label": "Choose my sources",
```
Comment (MaartenLMEM) [inline src/i18n/resources/en/extension.json:138]: Choose my contributors

File: src/i18n/resources/fr/extension.json [hunk 2]
```diff
@@ -133,5 +133,14 @@
     "tos": "/cgu",
     "about": "/a-propos",
     "uninstall": "/desinstaller"
+  },
+  "services": {
+    "label": "Choisir mes sources",
```
Comment (MaartenLMEM) [inline src/i18n/resources/fr/extension.json:138]: Choisir mes contributeurs

**Review by MaartenLMEM — CHANGES_REQUESTED** [review]

**Comment by MaartenLMEM** [comment]
just a little change in wording, all right to merge after

**Review by MaartenLMEM — APPROVED** [review]

**Pull request merged**